src/main/java/Duke.java

```diff
@@ -1,10 +1,76 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    final static String HORIZONTAL_LINE = "____________________________________________________________";
+
+    public static void greeting() {
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println(HORIZONTAL_LINE + '\n');
+
+        return;
+    }
+
+    public static void exit() {
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println(HORIZONTAL_LINE + '\n');
+
+        return;
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        ArrayList<Task> tasks = new ArrayList<Task>();
+
+        greeting();
+        Scanner sc = new Scanner(System.in);
+
+        while (true) {
```

Comment:
perhaps you could use a boolean variable in your conditional statement for your while loop?

personally I would use `isRunning`